dev-libs/wlroots/wlroots-0.1.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+if [[ ${PV} == 9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/swaywm/wlroots.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/swaywm/wlroots/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+inherit fcaps meson
+
+DESCRIPTION="Pluggable, composable, unopinionated modules for building a Wayland compositor"
+HOMEPAGE="https://github.com/swaywm/wlroots"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="elogind icccm rootston systemd x11-backend xkb X"
+REQUIRED_USE="?? ( elogind systemd )"
+
+RDEPEND=">=dev-libs/libinput-1.7.0:0=
+	>=dev-libs/wayland-1.16.0
+	>=dev-libs/wayland-protocols-1.15
+	media-libs/mesa[egl,gles2,gbm]
+	virtual/libudev
+	x11-libs/libdrm
+	x11-libs/libxkbcommon
+	x11-libs/pixman
+	elogind? ( >=sys-auth/elogind-237 )
+	icccm? ( x11-libs/xcb-util-wm )
+	systemd? ( >=sys-apps/systemd-237 )
+	x11-backend? ( x11-libs/libxcb:0= )
+	xkb? ( x11-libs/libxcb[xkb] )
+	X? (
+		x11-base/xorg-server[wayland]
+		x11-libs/libxcb:0=
+	)"
+DEPEND="${RDEPEND}"
+BDEPEND=">=dev-util/meson-0.48
+	virtual/pkgconfig"
+
+FILECAPS=( cap_sys_admin usr/bin/rootston )
+
+src_configure() {
+	# xcb-util-errors is not on Gentoo Repository (and upstream seems inactive?)
+	local emesonargs=(
+		"-Dxcb-errors=disabled"
+		-Dlibcap=$(usex filecaps enabled disabled)
+		-Dxcb-icccm=$(usex icccm enabled disabled)
+		-Dxcb-xkb=$(usex xkb enabled disabled)
+		-Dxwayland=$(usex X enabled disabled)
+		-Dx11-backend=$(usex x11-backend enabled disabled)
+		$(meson_use rootston)
```

Comment:
Please settle either on usex or meson_use. It makes no sense to use the other for one option.